webpack.config.js

```diff
@@ -92,6 +92,7 @@ const webpackConfig = ( mode ) => {
 				// New Modules (Post-JSR).
 				'googlesitekit-api': './assets/js/googlesitekit-api.js',
 				'googlesitekit-data': './assets/js/googlesitekit-data.js',
+				'googlesitekit-user': './assets/js/googlesitekit-user.js',
```

Comment:
This should be 
```suggestion
				'googlesitekit-user': './assets/js/googlesitekit-datastore-user.js',
```

Reply:
I find it awkward that we're scoping it in one place but not another. Why even have `googlesitekit-user` be it's own entrypoint, come to think of it? When isn't user info required when interacting with the API?

Reply:
Agreed, this should be similar to `googlesitekit-datastore-site` in the way it is approached regarding naming, file structure etc.